src/components/WebChat/index.jsx

```diff
@@ -156,6 +159,25 @@ class WebChat extends React.Component {
     }
   }
 
+  setIsRecording = (isRecording) => {
+    this.setState({ isRecording });
+  };
+
+  async setTranscript(transcript) {
+    this.setState({
+      transcript,
+    });
+
+    if (transcript) {
+      await this.sendMessage(transcript);
+      this.setState({
```

Comment:
If transcript, you end up setting it twice in a row. It is necessary ?

Reply:
No, you're right, it's old code I forgot to remove